---
channelCategory: Website
points:
  food: 4
  retail: 6
  services: 6
  other: 5
---

# Title contains business name and suburb/city

Your website's title tag is crucial for local SEO - we check if it includes both your business name and location to help you rank for local searches.

::tech-detail{summary="How we analyze your title tag"}
We extract your website's `<title>` tag and check for:
1. **Business name** - Exact or close match to your registered name
2. **Location** - Suburb, city, state, or area identifier

We cross-reference with your Google Business Profile location data when available. The title tag appears in browser tabs and as the blue clickable link in Google search results. It's one of the strongest on-page SEO signals.
::

## What we're checking

We verify that your website's title tag properly identifies both what your business is and where it's located, which is essential for appearing in "near me" searches and local results.

::impact{type="customers" severity="high"}
**46% of all Google searches** have local intent, and proper title tags can improve your local rankings by up to **20%**
::

## What issues may it cause?

### Invisible in local searches
- Miss out on "[service] near me" searches
- Don't appear for "[service] in [city]" queries
- Competitors with location in title rank above you
- Google can't determine your service area

### Lost click-through rates
- Generic titles get 50% fewer clicks
- Users can't tell if you serve their area
- No differentiation from competitors
- Appear less relevant in search results

### Wasted SEO potential
- Missing crucial ranking signal for local search
- Google may show wrong location information
- Can't compete for location-based keywords
- Reduced visibility in map pack results

::example{type="bad" title="Poor title tag impact"}
"Smith & Sons" used only their business name as the title:
- Invisible for "plumber Sydney" searches
- Lost 70% of potential local traffic
- Competitors "24/7 Plumber Sydney - Jones Plumbing" dominated
- When they added location: 
  - 185% increase in organic traffic in 3 months
  - Phone calls from website tripled
  - Ranked #2 for "emergency plumber Sydney"
::

## How can I fix it?

::fix-step{number="1" title="Analyze your current title"}
1. Google your business name
2. Look at the blue link text - that's your title
3. Check if it includes your location
4. View competitors' titles for comparison
5. Note your current search ranking position
::

::fix-step{number="2" title="Craft the perfect title"}
Use this proven formula:
```
[Service] [Location] - [Business Name] | [Tagline/Differentiator]
```

Examples:
- "Plumber Sydney CBD - Smith & Sons | 24/7 Emergency Service"
- "Italian Restaurant Bondi Beach - Luigi's | Authentic Wood Fired Pizza"
- "Tax Accountant Melbourne - Johnson CPA | Small Business Specialist"

Keep it under 60 characters for full display in Google.
::

::fix-step{number="3" title="Update your website"}
Where to change your title:
- **WordPress**: Yoast SEO or All-in-One SEO plugin
- **Wix/Squarespace**: Page settings > SEO
- **Custom HTML**: Edit `<title>` tag in header
- **Shopify**: Online Store > Preferences > Title

Don't forget to update:
- Homepage title (most important)
- Service pages with specific locations
- Contact page with full address area
::

::fix-step{number="4" title="Optimize for multiple locations"}
If you serve multiple areas:
- Homepage: Primary location only
- Create area pages: "Plumber North Sydney", "Plumber Chatswood"
- Use schema markup for service areas
- Consistent NAP (Name, Address, Phone) across all pages
::

::time-estimate{minutes="30" difficulty="easy"}
Research and planning: 15 minutes
Title tag updates: 10-15 minutes
Google re-indexing: 1-3 weeks
::

## What is the positive impact?

### Immediate search visibility
- **30-50% increase in local search impressions**
- Appear for location-specific searches
- Higher rankings for "near me" queries
- Better visibility in Google Maps

### Increased click-through rates
- Clear location relevance increases clicks by 25%
- Users know you serve their area
- Stand out from generic competitors
- Build trust before they visit your site

### Long-term SEO benefits
- Stronger local ranking signals
- Better Google My Business integration
- Compound effect with other local SEO
- Foundation for location page strategy

::example{type="good" title="Title optimization success"}
"Fresh Flowers" florist updated to "Florist Brisbane CBD - Fresh Flowers | Same Day Delivery":
- Week 1: Appeared on page 2 for "florist Brisbane"
- Month 1: Reached #4 for target keywords
- Month 3: #1 for "florist Brisbane CBD", #2 for "Brisbane florist"
- Results:
  - 420% increase in organic traffic
  - 65% of new customers mentioned finding them on Google
  - Opened second location based on search demand data
  - Valentine's Day sales up 250% year-over-year
::

## Learn more

- [Google's title tag guidelines](https://developers.google.com/search/docs/appearance/title-link) - Official best practices
- [Local SEO guide](https://moz.com/learn/seo/local) - Comprehensive local ranking factors
- [Title tag optimization](https://ahrefs.com/blog/title-tag-seo/) - Advanced techniques
- [Schema markup for local](https://schema.org/LocalBusiness) - Enhanced local presence 