---
channelCategory: Website
points:
  food: 2
  retail: 2
businessCategories:
  - food
  - retail
---

# Opening hours printed on the website

Customers need to know when you're open - we check if your website clearly shows your business hours so people know when they can visit or call.

::tech-detail{summary="How we detect opening hours on websites"}
Our system scans your website using multiple methods:
1. **Text patterns** - Looking for times like "9am-5pm" or "Monday-Friday"
2. **Common locations** - Checking headers, footers, contact pages, and "Hours" sections
3. **Structured data** - Reading schema.org markup that helps search engines
4. **Multiple formats** - Recognizing "24/7", "Open 7 days", "Closed Sundays", etc.

We use advanced pattern matching to find hours even if they're formatted unusually, and we filter out code or scripts that might contain time-like patterns.
::

## What we're checking

We scan your entire website to find where you've listed your opening hours. Just like a physical store needs a sign showing when it's open, your website needs this information easily visible to visitors.

::impact{type="customers" severity="medium"}
**67% of customers** check business hours online before visiting - if they can't find them, they might assume you're closed
::

## What issues may it cause?

### Wasted customer trips
- People drive to your location when you're closed
- Frustrated customers who made unnecessary trips rarely return
- Negative reviews mentioning "they were closed when website said open"

### Lost phone calls and sales  
- Customers calling outside hours get no answer
- They assume you're permanently closed or unreliable
- **42% won't try calling again** - they'll call a competitor instead

### Reduced trust
- Professional businesses display hours prominently
- Missing hours makes you seem disorganized
- Customers question if other information is current

::example{type="bad" title="Hidden hours hurt business"}
A retail shop had hours only on their Facebook page, not their website. Customer survey revealed:
- 3-5 people per day arrived when closed
- 18 one-star reviews mentioning "waste of time - they were closed"
- Estimated 20% drop in foot traffic over 6 months
::

## How can I fix it?

::fix-step{number="1" title="Add hours to multiple locations"}
Display your hours in at least 2-3 places:
- **Header or navigation** - Always visible
- **Footer** - Standard location customers check
- **Contact page** - Where people look for info
- **Homepage** - For maximum visibility
::

::fix-step{number="2" title="Use clear, standard formatting"}
Make hours easy to read and understand:
```
Monday - Friday: 9:00 AM - 6:00 PM
Saturday: 10:00 AM - 4:00 PM  
Sunday: Closed

Public Holidays: Closed
```
::

::fix-step{number="3" title="Add special circumstances"}
Include important details like:
- Holiday hours or closures
- Lunch breaks (if you close midday)
- Seasonal changes
- "Last order" times for restaurants
- Different hours for different services (e.g., "Kitchen closes at 9 PM")
::

::fix-step{number="4" title="Make it search-engine friendly"}
Help Google understand your hours:
- Use consistent formatting
- Include the words "hours" or "opening hours" near the times
- Consider adding schema.org markup (your web developer can help)
- Keep hours as text, not just in images
::

::time-estimate{minutes="15" difficulty="easy"}
Adding basic hours text: 15 minutes
Full implementation with schema markup: 30-45 minutes
::

## What is the positive impact?

### Happier customers
- No more wasted trips or frustrated visitors
- Customers can plan visits confidently
- Positive perception of your professionalism

### More business
- **23% increase in weekday visits** when hours are clearly shown
- Customers call during business hours when you can answer
- Reduced "where are your hours?" inquiries

### Better search visibility
- Google can show your hours directly in search results
- Improved local SEO rankings
- Hours appear in map listings automatically

::example{type="good" title="Clear hours drive visits"}
A bakery added prominent hours to their website header and footer:
- Customer complaints about closed doors dropped to zero
- 15% increase in morning customers (people could see they opened at 6 AM)
- Google started showing hours in search results within 2 weeks
- Phone calls during business hours increased by 30%
::

## Learn more

- [Schema.org markup for business hours](https://schema.org/openingHours) - Technical implementation
- [Google's guide to business hours](https://support.google.com/business/answer/3370250) - Best practices
- [Examples of effective hours display](https://www.nngroup.com/articles/contact-us-pages/) - UX research on contact information 